cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -265,8 +265,13 @@ function(px4_nuttx_add_export)
 		COMMAND ${MKDIR} -p ${nuttx_src}
 		COMMAND ${CP} -a ${CMAKE_SOURCE_DIR}/NuttX/. ${nuttx_src}/
 		COMMAND ${RM} -rf ${nuttx_src}/.git
+						  ${nuttx_src}/apps/.git
```

Comment:
@davids5 I need to test on windows, but I think rsync will work.
https://github.com/dagar/Firmware/commit/734c2c406db65d38efeb1ac57c0af77168bcec94